app/templates/gentelella/index.html

```diff
@@ -96,7 +96,46 @@
     <a href="/e/{{ event.identifier }}">
         <div class="col-md-4 col-xs-12 col-sm-6 event-holder">
             <div class="thumbnail">
-                <img src='{% if event.background_url%}{{ event.thumbnail }}{% elif order.event.topic%}{% set vars = {"foo": False} %}{% for custom_image in custom_placeholder %}{% if custom_image.name == session.event.topic %}{{ custom_image.url }}{% if vars.update({"foo": True}) %} {% endif %}{% endif %}{% endfor %}{% if not vars.foo %}{{ url_for("static", filename="placeholders/" + placeholder_images[order.event.topic])}}{% endif %}{% elif order.event.subtopic %}{% set vars = {"foo": False} %}{% for custom_image in custom_placeholder %}{% if custom_image.name == session.event.subtopic %}{{ custom_image.url }}{% if vars.update({"foo": True}) %} {% endif %}{% endif %}{% endfor %}{% if not vars.foo %}{ url_for("static", filename="placeholders/" + placeholder_images[order.event.subtopic])}}{% endif %}{% else %}{% set vars = {"foo": False} %}{% for custom_image in custom_placeholder %}{% if custom_image.name == "Other" %}{{ custom_image.url }}{% if vars.update({"foo": True}) %} {% endif %}{% endif %}{% endfor %}{% if not vars.foo %}{{ url_for("static", filename="placeholders/" + placeholder_images["Other"])}}{% endif %}{% endif %}' onerror="imgError(this, true)">
+                <img src='
+                {% if event.background_url%}
+                    {{ event.thumbnail }}
+                {% elif event.topic%}
+                {% set vars = {"foo": False} %}
```

Comment:
Could you name the variables more appropriately ? It would make it easier for future contributors to understand it's purpose.